### ISSUE TITLE
How to check generalization (validation loss) ?

### ISSUE THREAD

**Issue by 89douner** [tensorflow/models#4599]
### System information
-What is the top-level directory of the model you are using:  /models/research
-Have I written custom code: N/A
-OS Platform and Distribution: Linux Ubuntu 16.04
-TensorFlow installed from: pip installation
-TensorFlow version: Tensorflow: 1.5.0
-Bazel version: N/A
-CUDA/cuDNN version: CUDA:9.0/ cuDNN:7.1
- **GPU model and memory**: GeForce GTX 1070 Ti
- **Exact command to reproduce**:python3 train.py --logtostderr --train_dir=training/ --
### Describe the problem
When I train ssd_inception_v2 model, as below process, that message "global step~ loss ~" is showed on command screen. is the loss validation loss?
In addition, like below second image that shows a various type of loss, I couldn't know whether the loss is training loss or validation loss.
How could I check the validation loss of my training model??

### Source code / logs
![2018-06-22 12-26-51](https://user-images.githubusercontent.com/31752297/41756632-7a96bbf6-7618-11e8-9192-305377341bcf.png)
![2018-06-22 12-27-30](https://user-images.githubusercontent.com/31752297/41756638-7fd2a74c-7618-11e8-98fd-5f8d90410796.png)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version

**Comment by 89douner** [comment]
I updated the information you request.( But, I think that the environment information is not important to answer my question.)

**Comment by ykpgrr** [comment]
I am also wonder that where is the validation loss?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
